files/build_templates/docker_image_ctl.j2

```diff
@@ -326,6 +315,12 @@ start() {
     # Obtain our platform as we will mount directories with these names in each docker
     PLATFORM=${PLATFORM:-`$SONIC_CFGGEN -H -v DEVICE_METADATA.localhost.platform`}
 
+    # Default rsyslog target IP for single ASIC platform
+    SYSLOG_TARGET_IP=127.0.0.1
+    if [[ ($NUM_ASIC -gt 1) ]]; then
```

Comment:
This variable is undefined; it comes from `/usr/share/sonic/device/$PLATFORM/asic.conf`, which is sourced a few lines later. Because of this, on multi-ASIC systems, this condition will be false, and the syslog target IP will always be set to 127.0.0.1

Reply:
nice catch, fixed.